weed/command/mount.go

```diff
@@ -71,7 +71,7 @@ func init() {
 	mountOptions.debugPort = cmdMount.Flag.Int("debug.port", 6061, "http port for debugging")
 	mountOptions.localSocket = cmdMount.Flag.String("localSocket", "", "default to /tmp/seaweedfs-mount-<mount_dir_hash>.sock")
 	mountOptions.disableXAttr = cmdMount.Flag.Bool("disableXAttr", false, "disable xattr")
-	mountOptions.writeOnceReadMany = cmdMount.Flag.Bool("writeOnceReadMany", false, "write once, read many times")
+	mountOptions.worm = cmdMount.Flag.Bool("worm", false, "wrom mode, if true, you can change a file only once, and then it is readonly and undeletable, see https://en.wikipedia.org/wiki/Write_once_read_many")
```

Comment:
change to writeOnceReadMany here

Reply:
fix spelling error "wrom mode"

Reply:
"if true, you can change a file only once, and then it is readonly and undeletable"  simplify this.

Reply:
> "if true, you can change a file only once, and then it is readonly and undeletable" simplify this.

I think it is simple enough